cli/napi/node_api.rs

```diff
@@ -849,6 +820,15 @@ impl TsFn {
 
         let tsfn = unsafe { &*tsfn };
 
+        if tsfn.max_queue_size > 0 {
+          let mut queue_size = tsfn.queue_size.lock().unwrap();
+          let size = *queue_size;
+          *queue_size -= 1;
```

Comment:
is the queue_size ever decremented in the other part of the `is_closed.load(Ordering::Relaxed)` branch?

Reply:
`is_closing` prevents queuing more calls, so it doesn't need to do book-keeping anymore.